packages/components/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -222,7 +245,12 @@ export const ConfirmationModal = forwardRef(function ConfirmationModalInternal(
         onClick: handleAction("cancel"),
       }}
     >
-      <Content>{state.message && <Markdown content={state.message} />}</Content>
+      {state.message && (
+        <Content>
+          <Markdown content={state.message} />
+        </Content>
+      )}
+      {children}
```

Comment:
Following my comment above, I was wondering if we should do something like this
```suggestion
      {state.message ? (
        <Content>
          <Markdown content={state.message} />
        </Content>
      ) : children}
```